src/PantheonTerminalWindow.vala

```diff
@@ -325,6 +325,14 @@ namespace PantheonTerminal {
             default_width = PantheonTerminal.saved_state.window_width;
             default_height = PantheonTerminal.saved_state.window_height;
 
+            if (default_width == -1) {
+                default_width = Gdk.Screen.width () * 2 / 3;
```

Comment:
Is this HiDPI-safe? I.e. if I have a 3200x1800 display with a scaling factor of 2, does `Gdk.Screen.width()` return 3200 or 1600?